deploy/requirements.txt

```diff
@@ -0,0 +1,2 @@
+docker
```

Comment:
Would it be possible to use poetry here for the virtual environment? Just want to make sure we're standardized across the repo on these things.